### PR TITLE
update link and format

### PR DESCRIPTION
**Pull request by uiolee**
### Description

1. update some url. 
  use `main` replace `master`.
2. change some format.

### Motivation

1. `master` branch is deprecated.

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/api/index.md [hunk 1]
```diff
@@ -6,7 +6,16 @@ page-type: landing-page
...
+JavaScript APIs for WebExtensions can be used inside the extension's
+[background scripts](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#background_scripts)
+and in any other documents bundled with the extension,
+including [browser action](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_action)
+or [page action](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Page_actions) popups,
+[sidebars](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Sidebars),
+[options pages](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Options_pages),
+or [new tab pages](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/chrome_url_overrides).
+A few of these APIs can also be accessed by an extension's [content scripts](/en-US/docs/Mozilla/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#content_scripts).
+(See the [list in the content script guide](/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts#webextension_apis).)
```
Comment (Josh-Cena) [inline files/en-us/mozilla/add-ons/webextensions/api/index.md:18]: You don't need to manually wrap the lines; this makes it harder to read the diff.

Comment (uiolee) [reply files/en-us/mozilla/add-ons/webextensions/api/index.md]: ok, i revert it.

File: files/en-us/mozilla/add-ons/webextensions/examples/index.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ To try these examples, clone the repository then:
 
 1. load the extension from its source folder using the [Load Temporary Add-on](https://extensionworkshop.com/documentation/develop/temporary-installation-in-firefox/) feature. The extension stays loaded until you restart Firefox.
 2. open the extension's source folder at the command-line and use [`web-ext`](https://extensionworkshop.com/documentation/develop/getting-started-with-web-ext/) to run the extension. The extension stays loaded until you restart Firefox.
-3. in Firefox use **File** > **Open File** and find the example in the [`build`](https://github.com/mdn/webextensions-examples/tree/master/build) folder. The `build` folder contains built and signed versions of all the examples. This permanently installs the example.
+3. in Firefox use **File** > **Open File** and find the example in the [`build`](https://github.com/mdn/webextensions-examples/tree/main/build) folder. The `build` folder contains built and signed versions of all the examples. This permanently installs the example.
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/examples/index.md:19]: @uiolee I would recommend removing this line. We don't have a mechanism to automatically update these built versions of the examples and some are up to seven years old so I may not behave as expected. See https://github.com/mdn/webextensions-examples/issues/397

Comment (uiolee) [reply files/en-us/mozilla/add-ons/webextensions/examples/index.md:19]: I understand what you mean. But I think this is another job, maybe we should use another PR to deal with it.

**Review by rebloor — APPROVED** [review]
Thanks @uiolee – I would have deleted that line but fair enough.

**Pull request merged**